providers/fips/self_test_kats.c

```diff
@@ -745,13 +719,12 @@ int SELF_TEST_kats(OSSL_SELF_TEST *st, OSSL_LIB_CTX *libctx)
  */
 
 /*
- * The default private DRBG of the library context, saved for the duration
- * of KAT testing.
+ * Replacement "random" sources
+ * main_rand is used for most tests and it's set to generate mode.
```

Comment:
Do we need real entropy for Self tests?

Reply:
It's physically impossible to get it, so I sure hope not.

Reply:
As far as I could tell, it was being used for RSA blinding & I cannot see that requiring _proper_ entropy.